commands/logout.go

```diff
@@ -29,8 +30,9 @@ func runLogout(cmd *cobra.Command, args []string) error {
 	if len(gateway) == 0 {
 		return fmt.Errorf("gateway cannot be an empty string")
 	}
+	gateway = strings.TrimSpace(gateway)
```

Comment:
Would it make sense for the trim come before the length check, or am I thinking too defensively?

If it does, then this may make sense in other commands.

Reply:
I like the suggestion, could it be worth raising separately from this PR as an issue?

Reply:
Yes we should trim spaces before the length check. I will create an issue to track this.